source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -81,7 +81,7 @@ void BioXASAppController::onUserConfigurationLoadedFromDb()
 
 				foreach (AMRegionOfInterest *region, userConfiguration_->regionsOfInterest()){
 					if (!containsRegionOfInterest(geDetector->regionsOfInterest(), region)) {
-						geDetector->addRegionOfInterest(region->createCopy());
```

Comment:
the default behavior of `addRegionOfIntere()` is to create a copy or the `region`. Therefore, we won't have to force a copy here. Otherwise, it will be a potential memory leak.